.bettercodehub.yml

```diff
@@ -0,0 +1,8 @@
+component_depth: 1
```

Comment:
We avoid dependencies on external, paid, services like this. Please remove from the PR.